src/kernels/jupyter/jupyterUriProviderWrapper.ts

```diff
@@ -43,6 +43,9 @@ export class JupyterUriProviderWrapper implements IJupyterUriProvider {
     public get displayName(): string | undefined {
         return this.provider.displayName;
     }
+    public get detail(): string | undefined {
```

Comment:
`id` is not sufficient for the new quick pick, thus we introduced displayname and detail. We will need to revisit the updated API soon.